### ISSUE TITLE
FYI and Collaboration?

### ISSUE THREAD

**Issue by doowb** [digitaljohn/grunt-modulus-deploy#1]
I'm doing something similar as an express middleware with [this repo](https://github.com/upstage/continuous-deployment-modulus). Check it out and see if there's any code you could use.

I'd be interested in creating a library that could be used within both of these libraries if you'd like to collaborate.

**Comment by digitaljohn** [comment]
For sure. I'm still getting my head around custom grunt tasks though. :)

Do you want to setup a new non project in GitHub and add me as a collaborator? This new module is purely to trigger modulus deploys?

**Comment by doowb** [comment]
Yeah, that's all it would be for... Actually, it might not be needed now. I'll have to look into the modulus cli code and see if it's easier to require that in than it was before. If so, then we might be able to just use that inside our different projects.
